test/sample-app/package.json

```diff
@@ -7,7 +7,7 @@
         "@types/node": "^16.18.54",
         "@types/react": "^18.2.23",
         "@types/react-dom": "^18.2.8",
-        "@whereby.com/browser-sdk": "2.0.0-alpha19",
+        "@whereby.com/browser-sdk": "2.0.0-alpha26",
```

Comment:
Could you update the version in the postinstall script? (`test/sample-app/build-browser-sdk.sh`)

Reply:
Done 👍